Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue7015.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+
+namespace Xamarin.Forms.Controls.Issues
+{
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Github, 7015, "Image placeholder", PlatformAffected.All)]
+	public class Issue7015 : TestContentPage
+	{
+		Image image;
+		ImageButton imageButton;
+
+		//string url = "https://cdn.wallpaperhub.app/cloudcache/0/1/7/7/1/4/0177141d23e1d77489fa02a20f66302fe01033fd.jpg";
+		string url = "https://cdn.wallpaperhub.app/cloudcache/0/1/7/7/1/4/0177141d23e1d77489fa02a20f66302fe010fd.jpg";
+
+		string localImage = "bell.png";
+		string errorLocalImage = "games.png";
+
+		protected override void Init()
+		{
+			Padding = new Thickness(10);
+			image = new Image
+			{
+				Source = localImage,
+				LoadingSource = "bank.png",
+				ErrorSource = errorLocalImage
+			};
+
+			imageButton = new ImageButton
+			{
+				Source = localImage,
+				LoadingSource = "bank.png",
+				ErrorSource = errorLocalImage
+			};
+
+			var button = new Button
+			{
+				Text = "Change image",
+				VerticalOptions = LayoutOptions.CenterAndExpand,
+				BackgroundColor = Color.Fuchsia
+			};
+
+			button.Clicked += async (_, __) =>
+			{
+				var s = image.Source.ToString();
```

Comment:
Can you add descriptions here as I click the button so I know if what I'm seeing is the correct thing?

For example
"Image is being set to an invalid url so you should see a bank which is the error source"